lldb/include/lldb/Symbol/SymbolFile.h

```diff
@@ -425,6 +425,11 @@ class SymbolFile : public PluginInterface {
   /// Reset the statistics for the symbol file.
   virtual void ResetStatistics() {}
 
+  /// Get the number of loaded DWO files for this symbol file.
+  /// This is used for statistics gathering and will return 0 for most
+  /// symbol file implementations except DWARF symbol files.
+  virtual uint32_t GetLoadedDwoFileCount() const { return 0; }
```

Comment:
There is a `GetSeparateDebugInfo` member function that returns info about separate debug info (which dwo is one type). Looking [here](https://github.com/dmpots/llvm-project/blob/79108da325daec08f5b50169a9c35e03ea0645a3/lldb/source/Commands/CommandObjectTarget.cpp#L2650) it looks like we could use it to find all the dwos that were loaded.

I wonder if it would be better to use this existing function when generating the stats instead of adding a new overload here.